dxr/build.py

```diff
@@ -573,7 +576,7 @@ def index_chunk(tree,
                        open_log(tree.log_folder,
                                 'index-chunk-%s.log' % worker_number))
                 for path in paths:
-                    log and log.write('Starting %s.\n' % path)
+                    log and log.write('Starting %s.\n' % path.encode('utf-8'))
```

Comment:
It looks like it's actually the `print` statement that failed. I was going to suggest that the decoding error happens when the Python internals try to jam the Unicode output into an ASCII tty, but I see from the Travis logs that LANG=C.UTF-8. Can you try doing the encode just on the print statement and see what happens?